web/scss/components/facets.scss

```diff
@@ -99,6 +101,7 @@
   max-width: 100%;
   display: flex;
   flex-direction: column;
+  height: 100% !important;
```

Comment:
In general, we should try to avoid using `!important` as much as possible and consider it a last resort.  It usually just makes changes things later more difficult.